test/features/test_multiscale_feature_persistence.cpp

```diff
@@ -0,0 +1,191 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2021-, Open Perception
+ *
+ *  All rights reserved
+ */
+
+#include <pcl/common/norms.h>
+#include <pcl/features/fpfh.h>
+#include <pcl/features/multiscale_feature_persistence.h>
+#include <pcl/test/gtest.h>
+#include <pcl/point_cloud.h>
+
+using PointT = pcl::PointXYZ;
+using PointCloudT = pcl::PointCloud<PointT>;
+using FPFHSignature = pcl::FPFHSignature33;
+PointCloudT::Ptr cloud;
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+TEST(PCL, DistanceBetweenFeatures)
+{
+  pcl::detail::MultiscaleFeaturePersistenceTest<PointT, FPFHSignature> test_distance;
+  std::vector<float> a{101, -11, 24, 3, 18, 27, 65};
+  std::vector<float> b{89, 29, 24, 1008, -57, 106, 85};
```

Comment:
These values don't seem anything in particular. Is there a way to make the values seem less important, eg:
* by using randomly generated values
* or values with a pattern, eg: `1, 12, 23, 34, 45, ...`, `12, -34, 56, -78, ...`, etc.